jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java

```diff
@@ -6,31 +6,29 @@
 package io.opentelemetry.contrib.jfr.metrics.internal.container;
 
 import static io.opentelemetry.contrib.jfr.metrics.internal.Constants.ONE;
+import static io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler.defaultMeter;
 
 import io.opentelemetry.api.metrics.Meter;
-import io.opentelemetry.api.metrics.internal.NoopMeter;
 import io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler;
 import jdk.jfr.consumer.RecordedEvent;
 
 public final class ContainerConfigurationHandler implements RecordedEventHandler {
   private static final String EVENT_NAME = "jdk.ContainerConfiguration";
-  private static final String METRIC_NAME = "runtime.jvm.cpu.limit";
-
   private static final String EFFECTIVE_CPU_COUNT = "effectiveCpuCount";
 
   private volatile long value = 0L;
 
   public ContainerConfigurationHandler() {
-    initializeMeter(NoopMeter.getInstance());
+    initializeMeter(defaultMeter());
   }
 
   @Override
   public void initializeMeter(Meter meter) {
     meter
-        .upDownCounterBuilder(METRIC_NAME)
+        .upDownCounterBuilder("runtime.jvm.cpu.limit")
```

Comment:
Why are we inlining this constant? I thought the plan was to standardize on <OPTIONAL_PREFIX>_?METRIC_NAME so it's easy to grep for all of the generated metrics and which class handles them?

Reply:
> I thought the plan was to standardize on <OPTIONAL_PREFIX>_?METRIC_NAME so it's easy to grep for all of the generated metrics and which class handles them?

Ah.. I get why they're not inlined now. That makes sense to me.